test_proj/urls.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import url
 from django.contrib import admin
+from django.urls import re_path
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
+    re_path(r'^admin/', admin.site.urls),
```

Comment:
Wouldn’t it be simpler to use `path('admin/'` here?  a regex isn’t needed for this case.

Reply:
Sounds reasonable. Changed.